docs/release_notes/DRAFT_Release_notes_0.1.2.adoc

```diff
@@ -32,6 +32,8 @@ root package name to `de.adorsys.xs2a.adapter.impl`
 - Changed the root package of the `xs2a-adapter-service-api` module to `de.adapter.xs2a.adapter.api`.
 - Cleaned up package structure in `commerzbank-adapter`, `verlag-adapter` and `xs2a-adapter-service-remote`.
 - Removed `maven-adapter-archetype`.
+- Removed `PSU-ID` header from Payment Initiation call, and clear its value off from Consent Creation request for `consors-bank-adapter`,
```

Comment:
Why the behaviour is different for AIS and PIS?

Reply:
fixed